components/Ripple/src/MDCRippleTouchController.h

```diff
@@ -56,17 +56,45 @@
 @property(nonatomic, assign) BOOL shouldProcessRippleWithScrollViewGestures;
 
 /**
- Unavailable, please use `initWithView` instead.
+ Initializes the controller.
+
+ @param view The view that responds to the touch events for the ripple, and the view which the
+ ripple is added as a subview to.
  */
-- (nonnull instancetype)init NS_UNAVAILABLE;
+
+/**
+ Initializes the controller and adds the initialized ripple view as a subview of the provided view.
+
+ Note: When using this initializer, calling `addRippleToView:` isn't needed.
+
+ @param view The view that responds to the touch events for the ripple, and the view which the
+ ripple is added as a subview to.
+ @return a MDCRippleTouchController instance.
```

Comment:
Nit/spelling: Please use "an" instead of "a" because the letter "M" in "MDC" begins with a vowel sound ("em").

http://grammarguide.copydesk.org/2005/10/01/a-vs-an/

Reply:
Done